woocommerce/woocommerce-ios
# [Payments Menu] Prepare Payments Menu for release

Closes: #11111


## Description


This PR removes the toggle, feature flag, and code for the old payments menu, preparing for release of the rewritten SwiftUI payments menu.

Unit tests still need to be updated for the new menu, however, and have been temporarily commented out.

## Testing instructions


1. Delete the app from your test device
2. Launch the app from Xcode
3. Navigate to Menu > Payments
4. Observe that the new payments menu is shown


---
